include/brickengine/input.hpp

```diff
@@ -461,6 +474,19 @@ class BrickInput {
         keycode_mapping.insert({SDLK_PAGEUP, InputKeyCode::EKey_pageup});
         keycode_mapping.insert({SDLK_HOME, InputKeyCode::EKey_home});
 
+        keycode_mapping.insert({SDLK_F1, InputKeyCode::EKey_f1});
```

Comment:
To follow the previous 'inserted' keys, I would add the comment // F keys above this section of inserted F keys.

Reply:
Also agree with this